### PR TITLE
1873: Diesel fixes; More colors

### LINKED ISSUE #4865: [Harzbahn 1873] Need more colors for routes
Example game: https://18xx.games/game/33770?action=1689

Trying to create a 9th route results in this error: "undefined method `match' for nil"

I think we need at least 16 colors.

### LINKED ISSUE #4867: [Harzbahn 1873] Diesel route affecting revenue from normal trains
Game https://18xx.games/game/33770?action=1725

When the Diesel routes are added, the revenue produced by the "normal" trains drop by the amount the diesels are running for.

### PR DESCRIPTION
**Pull request by roseundy**
Touches assets/app/lib/settings.rb

Fixes #4865 
Fixes #4867 

Before:
colors_before

After:
colors_after

### CODE REVIEW

**Comment by yzemaze** [comment]
before is this:
image

(color 7 was changed recently #4561)

File: assets/app/lib/settings.rb [hunk 1]
```diff
@@ -6,8 +6,25 @@
...
+      #FFDC3D
+      #EF0096
+      #7CFFFA
+      #005FCC
+      #F60239
+      #00E307
+      #FFCFE2
+      #AFFF2A
+      #E8D2AB
+    ].freeze
```
Comment (yzemaze) [inline assets/app/lib/settings.rb:27]: hyperbole ;-) 2 lines would suffice.

Comment (roseundy) [reply assets/app/lib/settings.rb:27]: It was a lot easier to reorder this way.

**Comment by roseundy** [comment]
We already had enough pinks, I think.

File: assets/app/lib/settings.rb (hunk 1 above)
Comment (roseundy) [reply assets/app/lib/settings.rb]: OK, fixed.

**Comment by roseundy** [comment]
This shouldn't require pins.

**Pull request merged**